mottak/src/main/kotlin/no/nav/melosys/soknadmottak/mottak/MottakAdminController.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.melosys.soknadmottak.mottak
+
+import no.nav.melosys.soknadmottak.common.SikkerhetsbegrensningException
+import no.nav.security.token.support.core.api.Unprotected
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+private const val API_KEY_HEADER = "X-MELOSYS-ADMIN-APIKEY"
+
+@Unprotected
```

Comment:
Usikker på dette. I melosys-api er det ikke farlig å f.eks. gjenstarte en prosess som er ferdig. Her får du problemer i melosys-api inntil vi har idempotens der.

Reply:
Ja, det er et godt poeng. 🤔

Reply:
Sant det du sier Eivind, er nok ikke helt mulig å unngå duplikater. Man kan jo gjøre en del i produsenten for at duplikater ikke blir sendt, men vi har jo også opplevd generelt i NAV at det oppstod en feilsituasjon hvor meldinger ble lest på nytt av konsumenten... 
Så i bunn og grunn burde konsumenten kunne håndtere at samme melding blir sendt flere ganger. 

`soknadID` kunne f.eks. ha blitt lagret på behandlingsgrunnlaget? Som en "ekstern-referanse-ID" 🤷

Reply:
Jeg har oppdatert https://jira.adeo.no/browse/MELOSYS-4276